app/controllers/admin/users_controller.rb

```diff
@@ -1,27 +1,27 @@
+# frozen_string_literal: true
+
 class Admin::UsersController < Admin::AdminController
   active_scaffold :user do |config|
-    config.actions.exclude :create, :delete
-    config.columns = %I[email first_name last_name
+    config.actions.exclude :create, :delete, :search
+    config.columns = %I[email first_name last_name affiliate_names
```

Comment:
This introduces a new bug, as it changes the behavior of the users index page, by adding the "Affiliate names" column:

![image](https://user-images.githubusercontent.com/437455/154103570-a5f3dee4-c138-4e6d-a708-2c7a6efd243c.png)

We'll need to fix that and add a spec for the fix. (Note: the "Memberships" column is just a wordier version of the affiliates column, as it displays the affiliate name plus the user's email. I actually think the "Affiliate names" column is preferable, but I leave it up to you as to whether we want to chase down the desired UI behavior at this point in time.)

Reply:
Per the JIRA comment, I've proceeded with swapping out memberships for affiliate names.  This work is grouped in the final commit.